### PR TITLE
issue2632 Documentation contribution guidelines

### PR DESCRIPTION
**Pull request by stoobie**
RFE
This PR addresses issue #2632

Changes proposed in this pull request:

- Adding more detailed documentation contribution guidelines.

Preview: See the feature branch on my fork:
https://github.com/stoobie/ovirt-site/blob/RHV-43058.1/source/documentation/contributing_to_docs/contributing.adoc
https://github.com/stoobie/ovirt-site/blob/RHV-43058.1/source/documentation/contributing_to_docs/contributing_user_stories.adoc
https://github.com/stoobie/ovirt-site/blob/RHV-43058.1/CONTRIBUTING.md

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @stoobie

This pull request needs review by: @duck-rh @sandrobonazzola @janosdebugs

### CODE REVIEW

**Comment by stoobie** [comment]
@sandrobonazzola I want us to have some to prevent PRs from being merged before they're ready. Do you think we need WIP and the DO_NOT_MERGE label and to convert it to Draft? Do you think any one of those is enough? I think that marking it as Draft should be enough, right?

**Comment by janosdebugs** [comment]
@stoobie a draft PR is enough, that way it can't be merged.

**Comment by stoobie** [comment]
@emarcusRH @dcdacosta @RichardHoch Please review (and approve if it looks good).

File: CONTRIBUTING.md
Comment (RichardHoch) [inline CONTRIBUTING.md:31]: suggestion: remove "the"

File: CONTRIBUTING.md
Comment (RichardHoch) [inline CONTRIBUTING.md:33]: Suggestion: "Website content is..."

File: CONTRIBUTING.md
Comment (RichardHoch) [inline CONTRIBUTING.md:78]: Your change is fine but I have some suggestions/comments about the legacy text:
1 .commas after "sufficient"  and "To make an include".
2. Is "identically" the correct word? I can't think of a replacement, but it feels a bit "off" to me. If you can think of a better word, please use it.
3. Last sentence replace "if" with "it."  Suggestion "To make an include, create HAML content in .."

File: source/documentation/contributing_to_docs/contributing.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing.adoc:70]: Suggestions remove extra line (line 70). If this is supposed to mimic an onscreen message, add [source,terminal] to line 59 (that is the line above ----)

File: source/documentation/contributing_to_docs/contributing.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing.adoc:97]: suggestion "remains" -- it's a fact, so the present simple is accurate, as well as being preferred.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:8]: Do you need context information here?

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:11]: Suggestion: Either delete "backstory" or replace it with a phrase like "in oVirt and Red Hat Virtualization documentation".

File: source/documentation/contributing_to_docs/contributing.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing.adoc:70]: This file is markdown, not AsciiDoc, so that wouldn't work.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:22]: Recommendation: drop this section. It undercuts what we just said.  I suggest adding a sentence such as "To make a suggestion for improving existing content, reach out to the oVirt documentation team at <some link>." to the previosu section.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:14]: Suggestion: Add a brief description/definition of a user story -- with an example -- here.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:47]: Recommendation: "documentation team" instead of "docs team"

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:53]: Recommendation: This is an upstream document, so change lines 53 and 54 to "How will the end-user interact with this feature? Web console? CLI? REST API?"  [If the last two overlap, just use one of them].
Also eliminate "our" from line 52 unless it is appropriate for the upstream community.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:58]: Are these questions to ask community members?

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (RichardHoch) [inline source/documentation/contributing_to_docs/contributing_user_stories.adoc:65]: Can community members access this page?

**Comment by RichardHoch** [comment]
@stoobie -- I've left some comments, but otherwise, LGTM.

**Review by dcdacosta — COMMENTED** [review]
I also reviewed this content and beyond what Richard has suggested it LGTM.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:65]: @RichardHoch  The idea for anything on this site is that it's open and public.

File: CONTRIBUTING.md
Comment (stoobie) [reply CONTRIBUTING.md:31]: Fixed.

File: CONTRIBUTING.md
Comment (stoobie) [reply CONTRIBUTING.md:78]: 1. I left as-is.
2. I rewrote the sentence a bit.
3. Fixed.

File: source/documentation/contributing_to_docs/contributing.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing.adoc:97]: Fixed, but this section is commented out and only applies to OpenShift.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:22]: Well, if someone wants to make a contribution and sees this section, they might just skip the whole thing. Having the heading there make it more visible, I don't want to discourage someone.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:11]: Rewrote it.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc]: The existing example is very close in proximity to this sentence...do you really think that's necessary?

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:45]: Done.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:51]: Fixed.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:56]: Fixed.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:63]: Fixed.

File: source/documentation/contributing_to_docs/contributing_user_stories.adoc
Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:8]: Honestly, I'm not familiar with this attribute.

Comment (stoobie) [reply source/documentation/contributing_to_docs/contributing_user_stories.adoc:8]: I don't see a reason for it here.

**Comment by stoobie** [comment]
@dcdacosta  Can you please review my changes and if it looks good, can you approve so it can be merged?

**Review by dcdacosta — APPROVED** [review]

**Pull request merged**
